ckanext/resource_visibility/helpers.py

```diff
@@ -21,6 +16,7 @@ def _get_helpers():
         get_select_field_options,
         has_user_permission_for_org,
         get_assessment_result_help_url,
+        is_ckan_29
```

Comment:
Does this need to be a template helper? There's no template using it at this point.